staging/src/k8s.io/api/core/v1/annotation_key_constants.go

```diff
@@ -48,6 +48,22 @@ const (
 	// SeccompProfileRuntimeDefault represents the default seccomp profile used by container runtime.
 	SeccompProfileRuntimeDefault string = "runtime/default"
 
+	// AppArmorBetaContainerAnnotationKeyPrefix is the prefix to an annotation key specifying a container's apparmor profile.
+	AppArmorBetaContainerAnnotationKeyPrefix = "container.apparmor.security.beta.kubernetes.io/"
+	// AppArmorBetaDefaultProfileAnnotatoinKey is the annotation key specifying the default AppArmor profile.
+	AppArmorBetaDefaultProfileAnnotationKey = "apparmor.security.beta.kubernetes.io/defaultProfileName"
+	// AppArmorBetaAllowedProfileAnnotationKey is the annotation key specifying the allowed AppArmor profiles.
+	AppArmorBetaAllowedProfilesAnnotationKey = "apparmor.security.beta.kubernetes.io/allowedProfileNames"
+
+	// AppArmorProfileRuntimeDefault is the profile specifying the runtime default.
+	AppArmorProfileRuntimeDefault = "runtime/default"
```

Comment:
Maybe `AppArmorProfileRuntimeDefault`, `AppArmorProfileNamePrefix` and  `AppArmorProfileNameUnconfined` belong in a separate file, but kept it here to be consistent with `SeccompProfileRuntimeDefault`

Reply:
It's unclear these values will be used verbatim in the final promoted fields. Suggest naming the constants (and adding godoc) to indicate they pair with the beta annotations, e.g.:

```go
AppArmorBetaAnnotationProfileRuntimeDefault
AppArmorBetaAnnotationProfileNamePrefix
AppArmorBetaAnnotationProfileNameUnconfined
```

Reply:
done